drivers/radio/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+zephyr_sources_ifdef(CONFIG_RADIO_WCN14A2A
```

Comment:
The "radio" directory is quite generic, should we have "lte" or "modem" directory instead, what do people think about this?